src/DeviceInterface/InterfacePrototype.js

```diff
@@ -485,6 +498,12 @@ class InterfacePrototype {
     /** @returns {Promise<null|Record<any,any>>} */
     getUserData () { return Promise.resolve(null) }
 
+    /** @returns {void} */
+    removeUserData () {}
+
+    /** @returns {Promise<null|Record<string,boolean>>} */
+    getEmailProtectionCapabilities () { return Promise.resolve(null) }
```

Comment:
If this is a 'must implement on all devices' method, then I'd like us to start using the `throw new Error('unimplemented')` pattern in the base class here. This can help during development to prevent subtle bugs where someone accidentally removes the implementation from a subclass.

JavaScript is limited in what we can 'enforce' on implementors, this is a small alternative
```suggestion
    getEmailProtectionCapabilities () { throw new Error('unimplemented') }
```